core/src/trezor/messages.py

```
@@ -414,6 +414,138 @@ def __init__(
+    class BenfenAddress(protobuf.MessageType):
+        address: "str | None"
+
+        def __init__(
+            self,
+            *,
+            address: "str | None" = None,
+        ) -> None:
+            pass
+
+        @classmethod
+        def is_type_of(cls, msg: Any) -> TypeGuard["BenfenAddress"]:
```

Ruff (0.8.2)

Dynamically typed expressions (typing.Any) are disallowed in `msg`

(ANN401)